lib/shared/src/sourcegraph-api/graphql/client.ts

```diff
@@ -182,6 +183,27 @@ export interface ContextSearchResult {
     content: string
 }
 
+export interface ContextFiltersResult {
+    include: CodyContextFilterItem[]
+    exclude: CodyContextFilterItem[]
+}
+
+export interface CodyContextFilterItem {
+    repoNamePattern: string
+    // Not implemented
+    filePathPattern?: string
```

Comment:
```suggestion
    filePathPatterns?: string[]
```
In future iterations we plan to support multiple file path patterns per repo pattern ([RFC](https://docs.google.com/document/d/1PMeJHvY0ia2nMewFvs4vrFcmDbRDw9kSyljil-K8Pt8/edit#bookmark=id.hys3h7gd8w2a)).